beacon-chain/operations/slashings/service_new.go

```diff
@@ -0,0 +1,102 @@
+package slashings
+
+import (
+	"context"
+	"time"
+
+	"github.com/prysmaticlabs/prysm/v5/beacon-chain/startup"
+	"github.com/prysmaticlabs/prysm/v5/config/params"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	"github.com/prysmaticlabs/prysm/v5/time/slots"
+)
+
+// WithElectraTimer includes functional options for the blockchain service related to CLI flags.
+func WithElectraTimer(cw startup.ClockWaiter, currentSlotFn func() primitives.Slot) Option {
```

Comment:
The reason why the parameter is a func, and not a struct or interface, is because we would need to pass in the chain service, which creates a cyclic dependency that is very hard to fix.